mail_embed_image/__manifest__.py

```diff
@@ -11,6 +11,9 @@
     "depends": [
         'web',
     ],
+    "external_dependencies": [
+        'beautifulsoup4'
+    ],
```

Comment:
this should be a dictionary, cf https://www.odoo.com/documentation/14.0/developer/reference/addons/module.html